### PR TITLE
Code + with some outputs

### PR DESCRIPTION
**Pull request by KushanRaj**
"elise1" has the first 10 notes of Fur Elise
"test" is the test output we were supposed to run
"output" is another random output

### CODE REVIEW

**Comment by MukundVarmaT** [comment]
@KushanRaj Hey it looks really good man! Nice work. I like the idea of incorporating diversity using logits.

Hey, one last thing it would be great if you could add the notes of the vocab so that we can confirm on the cosine similarity and see if the embedding makes sense. 
Also, see if it makes some difference if you train it on all the songs in an epoch. (Just check this)

Once again COOL STUFF!

**Comment by MukundVarmaT** [comment]
Once you check those out, I think this is good to go.
